src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts

```diff
@@ -9,25 +9,26 @@ export default function useGetTrendingSoonTokenId(token?: Token): number | undef
 
   useEffect(() => {
     const asyncF = async () => {
-      setTokenId(undefined)
-      if (token) {
-        const { address } = token
-        const url24h = `${process.env.REACT_APP_TRUESIGHT_API}/api/v1/trending-soon?timeframe=24h&page_number=0&page_size=${TRENDING_SOON_MAX_ITEMS}&search_token_address=${address}`
-        // const url7d = `${process.env.REACT_APP_TRUESIGHT_API}/api/v1/trending-soon?timeframe=7d&page_number=0&page_size=${TRENDING_SOON_MAX_ITEMS}&search_token_address=${address}`
-        const responses = await Promise.all([fetch(url24h)])
-        for (let i = 0; i < responses.length; i++) {
-          const response = responses[i]
-          if (response.ok) {
-            const { data }: { data: TrueSightTokenResponse } = await response.json()
-            if (data.tokens.length) {
-              setTokenId(data.tokens[0].token_id)
-              return
+      try {
+        setTokenId(undefined)
+        if (token) {
+          const { address } = token
+          const url24h = `${process.env.REACT_APP_TRUESIGHT_API}/api/v1/trending-soon?timeframe=24h&page_number=0&page_size=${TRENDING_SOON_MAX_ITEMS}&search_token_address=${address}`
+          // const url7d = `${process.env.REACT_APP_TRUESIGHT_API}/api/v1/trending-soon?timeframe=7d&page_number=0&page_size=${TRENDING_SOON_MAX_ITEMS}&search_token_address=${address}`
```

Comment:
missed remove this?

Reply:
this file, I just wrap with try catch to prevent error.
No logic update here

Reply:
I don't know that author of that file will use in the future or not.